core/interfaces/rebase.py

```diff
@@ -301,13 +302,21 @@ def _get_diverged_outside_rebase(self):
     def render_conflicts_bindings(self):
         return self.conflicts_keybindings if self._in_rebase else ""
 
-    def base_ref(self):
+    def base_ref(self, reset_ref=False):
         base_ref = self.view.settings().get("git_savvy.rebase.base_ref")
 
-        if not base_ref:
+        if not base_ref or reset_ref:
             project_data = sublime.active_window().project_data() or {}
             project_settings = project_data.get('settings', {})
-            base_ref = project_settings.get("rebase_default_base_ref", "master")
+            base_ref = project_settings.get("rebase_default_base_ref")
+
+            if not base_ref:
+                # use remote tracking branch as a sane default
+                remote_branch = self.get_active_remote_branch().name_with_remote
```

Comment:
`self.get_active_remote_branch()` could be `None` if there is no remote tracking, you probably only need `get_upstream_for_active_branch()`.

Reply:
I have fixed it.